config/django/base.py

```diff
@@ -43,6 +43,9 @@
     'open_schools_platform.organization_management.employees.apps.EmployeesConfig',
     'open_schools_platform.errors.apps.ErrorsConfig',
     'open_schools_platform.testing_examples.apps.TestingExamplesConfig',
+    'open_schools_platform.parent_management.families.apps.FamiliesConfig',
+    'open_schools_platform.parent_management.parents.apps.ParentsConfig',
+    'open_schools_platform.student_management.student_profile.apps.StudentProfileConfig',
```

Comment:
App name should be Student. Not Profile Student. It is because ProfileStudent is small part of this app and soon we would create model Student that would have much more functionality than StudentProfile.